src/index.ts

```diff
@@ -0,0 +1 @@
+require("./cli/index.ts");
```

Comment:
we should `require("./cli");` here so the same call will work in both TS and JS.

Reply:
yep word.